packages/decorators/tests/djs/RequiresUserPermissions.test.ts

```diff
@@ -0,0 +1,115 @@
+import { UserError } from '@sapphire/framework';
+import { Message as DJSMessage, PermissionResolvable, Permissions, PermissionString } from 'discord.js';
+import diff from 'lodash/difference';
```

Comment:
could this use lodash.difference?

Reply:
We're already pulling in lodash as a transient dep anyway, so also pulling in `lodash.difference` actually adds more dependencies than using `lodash/difference`.

Reply:
```
dev/sapphiredev/utilities  feat/add-user-permissions-decorator (d57e0a4)
🕙 22:04 ❯ yarn why lodash
yarn why v1.22.5
[1/4] Why do we have the module "lodash"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "lodash@4.17.21"
info Has been hoisted to "lodash"
info Reasons this module exists
   - "workspace-aggregator-8693b9d5-438a-4d36-bfd3-921a835ced5f" depends on it
   - Specified in "devDependencies"
   - Hoisted from "_project_#ts-jest#lodash"
   - Hoisted from "_project_#@commitlint#cli#lodash"
   - Hoisted from "_project_#typedoc#lodash"
   - Hoisted from "_project_#lodash"
   - Hoisted from "_project_#@commitlint#config-conventional#conventional-changelog-conventionalcommits#lodash"
   - Hoisted from "_project_#cz-conventional-changelog#@commitlint#load#lodash"
   - Hoisted from "_project_#cz-conventional-changelog#commitizen#lodash"
   - Hoisted from "_project_#cz-conventional-changelog#commitizen#inquirer#lodash"
   - Hoisted from "_project_#@commitlint#cli#@commitlint#read#git-raw-commits#lodash"
   - Hoisted from "_project_#cz-conventional-changelog#@commitlint#load#@commitlint#resolve-extends#lodash"
   - Hoisted from "_project_#lerna#@lerna#create#whatwg-url#lodash"
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#conventional-commits#conventional-changelog-core#lodash"
   - Hoisted from "_project_#@commitlint#cli#@commitlint#lint#@commitlint#parse#conventional-commits-parser#lodash"
   - Hoisted from "_project_#@commitlint#cli#@commitlint#lint#@commitlint#rules#@commitlint#ensure#lodash"
   - Hoisted from "_project_#lerna#@lerna#clean#@lerna#prompt#inquirer#lodash"
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#conventional-commits#conventional-changelog-core#conventional-changelog-writer#lodash"
info Disk size without dependencies: "4.86MB"
info Disk size with unique dependencies: "4.86MB"
info Disk size with transitive dependencies: "4.86MB"
info Number of shared dependencies: 0
```